tools/claude-review/scripts/mdsafe.py

```
@@ -0,0 +1,165 @@
+    apply の対象として解釈するのは info string がちょうど "suggestion"
+    の場合のみなので、ここを崩してはならない。
+    """
+    runs = re.findall(r"`+", content)
+    longest = max((len(r) for r in runs), default=0)
+    return "`" * max(3, longest + 1)
+
+
+def code(s, table: bool = False) -> str:
+    """外部由来の短い文字列を、閉じられないインラインコードにする。
+
+    `esc()` はバッククォートに触れない（本文中の書式には手を出さない方針）。
```

Ruff (0.16.3)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)